docs/source/optional_plugins/varianter_cit.rst

```diff
@@ -0,0 +1,183 @@
+====================
+CIT Varianter Plugin
+====================
+
+This plugin is an implementation of a Combinatorial Interaction
+Testing algorithm for the Avocado varianter functionality.  It
+generates an optimal number of variants, which in turn become
+different test scenarios.
+
+.. note:: The publication by Ahmed, Bestoun S., Kamal Z. Zamli, and
+          Chee Peng Lim, entitled `“Application of particle swarm
+          optimization to uniform and variable strength covering array
+          construction” <http://booksc.org/book/13151468/aa9fc9>`__,
+          Applied Soft Computing, 12(4), 2012, pp. 1330-1347, contains
+          the basis for the algorithm and implementation of this
+          feature.
+
+Files
+=====
+
+- ``optional_plugins/varianter_cit/avocado_varianter_cit/__init__.py``: The
+  plugin implementation.
+
+- ``examples/varianter_cit/params.ini``: An example parameters file.
+
+Usage
+=====
+
+To see the variants generated by this demo implementation, execute::
+
+    $ avocado variants --cit-parameter-file examples/varianter_cit/params.ini
+    CIT Variants (14):
+    Variant gold-triangle-solid-aluminum-cathodic:    /run
+    Variant red-circle-gas-leather-cathodic:    /run
+    Variant gold-square-gas-plastic-anodic:    /run
+    Variant black-circle-liquid-aluminum-anodic:    /run
+    Variant green-triangle-liquid-leather-anodic:    /run
+    Variant green-circle-solid-plastic-cathodic:    /run
+    Variant black-square-solid-leather-cathodic:    /run
+    Variant red-square-liquid-aluminum-anodic:    /run
+    Variant red-triangle-liquid-plastic-cathodic:    /run
+    Variant black-triangle-gas-plastic-anodic:    /run
+    Variant green-square-gas-aluminum-anodic:    /run
+    Variant gold-circle-solid-leather-anodic:    /run
+    Variant gold-triangle-liquid-plastic-anodic:    /run
+    Variant red-circle-solid-aluminum-cathodic:    /run
+
+.. note:: The exact variants generated are not guaranteed to be the same
+          across executions.
```

Comment:
you make it sound as if that was an issue :-) I find that useful (and in some was better than pict).